tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs

```diff
@@ -18,20 +18,17 @@ namespace SixLabors.ImageSharp.Tests.Formats.Jpg
     [Trait("Format", "Jpg")]
     public class JpegColorConverterTests
     {
+        private static readonly float MaxColorChannelValue = 255f;
+
         private const float Precision = 0.1F / 255;
 
-        private static readonly ApproximateColorSpaceComparer ColorSpaceComparer = new ApproximateColorSpaceComparer(Precision);
+        private const int TestBufferLength = 40;
```

Comment:
We need to test remainder paths, which doesn't happen with `TestBufferLength=40`.

Reply:
Jpeg files always have strides of pixels divisible by 8, there will neven be a remainder problem. The only way remainder can exist - AVX-512 which has 16 floats per vector. `40 % 16 == 8`, size of 40 pixels covers possible remainder path.